src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,666 @@
+submariner-routeagent-wqmzs                    1/1     Running   0          75s
+```
+
+You can use this command to verify on which node the Gateway Engine is running on:
+
+```bash
+$ kubectl get node --selector=submariner.io/gateway=true -o wide
+NAME              STATUS   ROLES    AGE     VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE       KERNEL-VERSION           CONTAINER-RUNTIME
+cluster2-worker   Ready    <none>   6h59m   v1.17.0   172.17.0.7    <none>        Ubuntu 19.10   5.8.18-200.fc32.x86_64   containerd://1.3.2
+```
+
+You can use this command to quickly verify the connection between the participating clusters:
+<!-- markdownlint-disable no-trailing-spaces -->
+```bash
+$ subctl show connections
+
+Showing information for cluster "cluster2":
+GATEWAY                         CLUSTER                 REMOTE IP       CABLE DRIVER        SUBNETS                               STATUS
+cluster3-worker                 cluster3                172.17.0.10     libreswan           100.3.0.0/16, 10.3.0.0/16             connected
+```
+
+You can use this command to get detailed information about the inter-cluster connection:
+
+```bash
+$ kubectl -n submariner-operator describe Gateway
+Name:         cluster2-worker
+Namespace:    submariner-operator
+Labels:       <none>
+Annotations:  update-timestamp: 1606751397
+API Version:  submariner.io/v1
+Kind:         Gateway
+Metadata:
+  Creation Timestamp:  2020-11-30T13:51:39Z
+  Generation:          538
+  Resource Version:    28717
+  Self Link:           /apis/submariner.io/v1/namespaces/submariner-operator/gateways/cluster2-worker
+  UID:                 682f791a-00b5-4f51-8249-80c7c82c4bbf
+Status:
+  Connections:
+    Endpoint:
+      Backend:          libreswan
+      cable_name:       submariner-cable-cluster3-172-17-0-10
+      cluster_id:       cluster3
+      Health Check IP:  10.3.224.0
+      Hostname:         cluster3-worker
+      nat_enabled:      false
+      private_ip:       172.17.0.10
+      public_ip:        
+      Subnets:
+        100.3.0.0/16
+        10.3.0.0/16
+    Latency:
+      Average RTT:   472623
+      Last RTT:      583181
+      Max RTT:       2603444
+      Min RTT:       80202
+      Stddev RTT:    164687
+    Status:          connected
+    Status Message:  
+  Ha Status:         active
+  Local Endpoint:
+    Backend:          libreswan
+    cable_name:       submariner-cable-cluster2-172-17-0-7
+    cluster_id:       cluster2
+    Health Check IP:  10.2.224.0
+    Hostname:         cluster2-worker
+    nat_enabled:      false
+    private_ip:       172.17.0.7
+    public_ip:        
+    Subnets:
+      100.2.0.0/16
+      10.2.0.0/16
+  Status Failure:  
+  Version:         v0.8.0-pre0-1-g5d7f163
+Events:            <none>
+```
+<!-- markdownlint-enable no-trailing-spaces -->
+To verify that service discovery is installed properly, check that the `submariner-lighthouse-coredns` service is ready:
```

Comment:
Are you still on cluster2? Maybe add that again, it confuses me a little (maybe it's only me though :D)